image_explorer/templates/html/image_explorer.html

```diff
@@ -9,7 +9,7 @@ <h2>{{title}}</h2>
     <div class="image-explorer-wrapper">
         <img src="{{background.src}}" class="image-explorer-background" />
         {% for hotspot in hotspots %}
-        <div class="image-explorer-hotspot" style="position: absolute; top: {{hotspot.y}}; left: {{hotspot.x}};" data-item-id="{{hotspot.item_id}}">
+        <div class="image-explorer-hotspot {% if hot_spots %}latest{% endif %}" style="position: absolute; top: {{hotspot.y}}; left: {{hotspot.x}};" data-item-id="{{hotspot.item_id}}">
```

Comment:
Please use a more descriptive class name. I suggest `centered` in stead of `latest`.